packages/@aws-cdk/aws-codepipeline-actions/test/s3/test.s3-cache-control.ts

```diff
@@ -0,0 +1,36 @@
+import { Duration } from "@aws-cdk/core";
+import { Test } from 'nodeunit';
+import cpactions = require('../../lib');
+
+// tslint:disable:object-literal-key-quotes
+
+const testDuration = Duration.hours(1);
```

Comment:
Can you rename this to `oneHour`? `testDuration` doesn't tell me much when evaluating whether the behavior is correct...

Reply:
Done